Source/DafnyDriver/Commands/VerifyCommand.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.Dafny;
 public static class VerifyCommand {
 
   static VerifyCommand() {
+    // TODO: This should be recorded and checked somehow for the same reason --no-verify is
```

Comment:
TODO

Reply:
Realized this was safe after all and rewrote accordingly.